src/code.cloudfoundry.org/loggregator/trafficcontroller/internal/proxy/log_access_middleware.go

```diff
@@ -21,6 +22,12 @@ func NewLogAccessMiddleware(authorize auth.LogAccessAuthorizer) *LogAccessMiddle
 func (m *LogAccessMiddleware) Wrap(h http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		appID := mux.Vars(r)["appID"]
+		guid := regexp.MustCompile("^[[:xdigit:]]{8}-[[:xdigit:]]{4}-[[:xdigit:]]{4}-[[:xdigit:]]{4}-[[:xdigit:]]{12}$")
```

Comment:
Same request as the [PR for v89.x](https://github.com/cloudfoundry/loggregator-release/pull/364#pullrequestreview-105099596)